docs/src/TapArea.doc.js

```diff
@@ -220,26 +220,37 @@ card(
         defaultValue: 'null',
         description: [
           'Define the frame or window to open the anchor defined on `href`:',
-          '- `null` opens the anchor in the same window.',
-          '- `blank` opens the anchor in a new window.',
-          '- `self` opens an anchor in the same frame.',
+          `- 'null' opens the anchor in the same window.`,
+          `- 'blank' opens the anchor in a new window.`,
+          `- 'self' opens an anchor in the same frame.`,
           'Optional with role=link.',
         ],
         href: 'roles',
       },
+      {
+        name: 'tapStyle',
+        type: `none | 'compress'`,
+        required: false,
+        defaultValue: 'none',
+        description: [
+          'Set a compressing behavior when the TapArea is clicked / touched.',
+          `- 'none' does not compress TapArea.`,
+          `- 'compress' scales down TapArea by 1%.`,
```

Comment:
@AlbertCarreras I would remove the `by 1%`

We actually compress it this way: https://github.com/pinterest/gestalt/blob/4f520db60d005670c8db7b6bee1500fa2e1a0419/packages/gestalt/src/useTapFeedback.js#L48

The reason for doing that is so we do not shrink large objects too much

Reply:
DONE